docs/docker.md

```diff
@@ -20,11 +20,11 @@ The following steps can be used to setup Stringer on Docker, using a Postgres da
 docker run --detach \
     --name stringer-postgres \
     --restart always \
-    --volume /srv/stringer/data:/var/lib/postgresql/data \
+    --volume ~/stringer:/var/lib/postgresql/data \
```

Comment:
I honestly don't know anything about best practices with this sort of thing. Can you help me understand what the difference is here, and why this is an improvement?

Reply:
I've reverted this change. I don't think the postgres data location on the host machine really matters; as long as it's somewhere specific to stringer that won't get accidentally overwritten/lost, it should be fine. I think a location in `~` is too easy to accidentally remove, so I think we're better served by keeping the `/srv` subdir. I've also seen usage of the `/opt` directory, but again, I think this is just personal choice.